src/seml/commands/start.py

```diff
@@ -746,7 +757,7 @@ def start_local_worker(
                 if (
                     debug_server
                     or output_to_console
-                    or logging.root.level <= logging.VERBOSE
```

Comment:
Same thing, probably not relevant since you most likely have some rationale for switching levels...